draft-irtf-cfrg-aegis-aead.md

```diff
@@ -725,15 +725,15 @@ return tag
 
 AEGIS-256 has a 768-bit state, made of six 128-bit blocks `{S0, ...S5}`.
 
-The parameters for this algorithm, whose meaning is defined in {{!RFC5116, Section 4}} are:
+The parameters for this algorithm, whose meaning is defined in {{!RFC5116, Section 4}}, are:
 
 - `K_LEN` (key length) is 32 bytes (256 bits).
 - `P_MAX` (maximum length of the plaintext) is 2<sup>61</sup> - 1 bytes (2<sup>64</sup> - 8 bits).
-- `A_MAX` (maximum length of the associated data) is 2<sup>61</sup> bytes (2<sup>64</sup> bits).
+- `A_MAX` (maximum length of the associated data) is 2<sup>61</sup> - 1 bytes (2<sup>64</sup> - 8 bits).
 - `N_MIN` (minimum nonce length) = `N_MAX` (maximum nonce length) = 32 bytes (256 bits).
 - `C_MAX` (maximum ciphertext length) = `P_MAX` + tag length = (2<sup>61</sup> - 1) + 16 or 32 bytes (in bits: (2<sup>64</sup> - 8) + 128 or 256 bits).
 
-Distinct associated data inputs, as described in {{!RFC5116, Section 3}} shall be unambiguously encoded as a single input.
+Distinct associated data inputs, as described in {{!RFC5116, Section 3}} MUST be unambiguously encoded as a single input.
```

Comment:
Could add a comma after `Section 3}}` or rephrase to `single input, as described...`.

Reply:
These bits could be moved to the Authenticated Encryption `Security:` section.

Reply:
Added a comma. We shouldn't move things unless really necessary.